### PR TITLE
Markdown sample

### PR DESCRIPTION
**Pull request by ehoogerbeets**
- includes frontmatter localization

### CODE REVIEW

File: markdown/project.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
+{
+    "name": "sample-md",
+    "id": "sample-md",
+    "projectType": "custom",
+    "sourceLocale": "en-US",
+    "pseudoLocale": "zxx-XX",
+    "resourceDirs": {
+        "java":"res"
```
Comment (gouniLee) [inline markdown/project.json:8]: I think it doesn't need it here.

**Review by gouniLee — COMMENTED** [review]
I manually updated ko-KR xliff file. and looks like it doesn't localize properly. Could you please check?

**Comment by ehoogerbeets** [comment]
The ko-KR translations needed a target-language attribute

File: markdown/xliffs/sample-md-nl-NL.xliff [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+<xliff version="1.2">
+  <file original="md/en-US/README.md" source-language="en-US" target-language="nl-NL" product-name="sample-md">
+    <body>
+      <trans-unit id="1" resname="Title" restype="string" datatype="x-yaml">
+        <source>Readme for the Github-Flavored Markdown Sample</source>
+        <target>Readme voor het Github-flavored markdown voorbeeld</target>
+      </trans-unit>
+    </body>
+  </file>
+  <file original="md/en-US/README.md" source-language="en-US" target-language="nl-NL" product-name="sample-md">
```
Comment (gouniLee) [inline markdown/xliffs/sample-md-nl-NL.xliff:11]: Is there a specific reason to put into two file tags? All data is from the same file. I think it should be in the same file tag. isn't it?

Comment (ehoogerbeets) [reply markdown/xliffs/sample-md-nl-NL.xliff]: I merged them. Originally there was a bug in ghfm where the "original" tag was 2 different things for the frontmatter and the body of the markdown. When I fixed that, I updated the original and didn't think about making the two "file" tags into one.

**Review by gouniLee — APPROVED** [review]

**Pull request merged**